addons/sale/static/tests/sales_team_dashboard_tests.js

```diff
@@ -59,10 +59,10 @@ QUnit.test("edit target with several o_kanban_primary_bottom divs", async (asser
             <kanban>
                 <field name="invoiced_target"/>
                 <templates>
-                    <div t-name="kanban-box" class="container o_kanban_card_content">
+                    <div t-name="kanban-card">
```

Comment:
I don't know if this test still makes sense if you change that. Maybe it is better to let the test as it is for now